packages/opentelemetry-basic-tracer/src/BasicTracer.ts

```diff
@@ -35,6 +35,7 @@ import { mergeConfig } from './utility';
 import { SpanProcessor } from './SpanProcessor';
 import { NoopSpanProcessor } from './NoopSpanProcessor';
 import { MultiSpanProcessor } from './MultiSpanProcessor';
+import { defaultConfig } from './config';
```

Comment:
Since `defaultConfig` is a shared immutable constant, should it be DEFAULT_CONFIG in all caps?